cmd/kubeadm/app/constants/constants.go

```diff
@@ -189,6 +189,9 @@ const (
 	// KubeConfigVolumeName specifies the name for the Volume that is used for injecting the kubeconfig to talk securely to the api server for a control plane component if applicable
 	KubeConfigVolumeName = "kubeconfig"
 
+	// KubeCloudConfigVolumeName specifies the name for the Volume that is used to inject the cloud configuration for kube-apiserver and kube-controller-manager component
+	KubeCloudConfigVolumeName = "kube-cloud-config"
```

Comment:
please define the constant in the same `volumes.go` file as I did in https://github.com/kubernetes/kubernetes/pull/56513

Reply:
Ack on it